.github/workflows/update-langs.yaml

```diff
@@ -0,0 +1,30 @@
+name: Update supported languages.
+on:
+  schedule:
+    - cron: "0 0 */30 * *"
+
+jobs:
+  updateLanguages:
+    if: github.repository == 'anuraghazra/github-readme-stats'
+    name: Update supported languages.
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci
+        env:
+          CI: true
+
+      - name: Run update-languages-json.js script
+        run: npm run generate-langs-json
```

Comment:
This will just run the script, generate the json file and write it to the GitHub action's cloned env. 

This won't commit or open a PR to the master branch.

Reply:
Good catch in my hurries; I forgot the most important thing. 😅 Fixed with 2c5fe3ed27f8a5926acb61fe6a552b3fe36970dd.